fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/ElectionQuestion.java

```diff
@@ -1,5 +1,4 @@
-package com.github.dedis.student20_pop.model.network.method.message.data.election;
-
+package com.github.dedis.student20_pop.model.network.method.message;
```

Comment:
I think keeping this file in the package `data/election`is better

Reply:
I will change it right now